### PR TITLE
CRIMRE-73-copy-ref-num

### PR DESCRIPTION
**Pull request by willmcb**
## Description of change
Allows the user to copy the application reference number to the system clipboard

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-73

### Before changes:
image

### After changes:
image

## How to manually test the feature
- User logs into application
- User clicks on applicants name in the table as shown above in the screenshot in either Your list, All open applications or Closed applications pages.
- User is guided to the applicants page
- User clicks on the copy link to copy the applicants reference number
- User can then paste the applicants reference number by using ctrl+v onto another application or window to verify the feature is working

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -46,8 +46,8 @@
   config.include_context 'with a logged in user', type: :system
 
   # Use the faster rack test by default for system specs
-  config.before(:each, type: :system) do
-    driven_by :rack_test
+  config.before(:each, type: :system) do |example|
+    driven_by :rack_test if example.metadata.fetch(:rack_test, true)
```
Comment (timpeat) [inline spec/rails_helper.rb:50]: Probably best to discard this change now you are no longer making use of it.

Comment (willmcb) [reply spec/rails_helper.rb]: Done :)

**Comment by MohammedHassanLatif93** [comment]
@timpeat I've fixed all the issues and the tests have now passed. Please can you review? Thank you 👍

File: app/views/crime_applications/_review_overview.html.erb [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <p class="govuk-body-l govuk-!-margin-bottom-2">
-      <%= crime_application.reference %>
+      <span id="reference-number">
+        <%= crime_application.reference %>
+      </span>
+      <%= link_to 'Copy', '', id: 'copy-reference-number', class: 'govuk-link' %>
```
Comment (arthurashman) [inline app/views/crime_applications/_review_overview.html.erb:7]: class: 'govuk-link--no-visited-state'

Prevents it turning purple after it's clicked

Comment (MohammedHassanLatif93) [reply app/views/crime_applications/_review_overview.html.erb]: @arthurashman thank you I've added this into the code for the copy button 👍

**Review by timpeat — APPROVED** [review]

**Pull request merged**